app/controllers/departments_controller.rb

```diff
@@ -0,0 +1,58 @@
+class DepartmentsController < ApplicationController
+  include Constants
+
+  before_action :set_department, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @head_of_department = HEAD_OF_DEPARTMENT
+    @departments = Department.all
+  end
+
+  def new
+    @department = Department.new
+  end
+
+  def create
+    @department = Department.new(department_params)
```

Comment:
`department = Department.new(department_params)` should be enough. no need for a class var

Reply:
and here as well(